### PR TITLE
Updating raspberry pi setup instructions

### PR DESCRIPTION
**Pull request by danamlewis**
Includes:
1. Minor edits and revisions of links, language, etc.
2. Added section (3b) how to do multiple wifi setup (for hotspot and home, suggested) from a Mac
3. Added note about console cable and good how-to link for using that with Mac
4. Edited watchdog section to run now, w/ @channemann's notes from his gist on that step

### CODE REVIEW

**Comment by diabeticgonewild** [comment]
This looks so awesome! :-) Thank you so much for polishing and making the install instructions Bette!

Because my initial install instructions had *.lan instead of the correct *.local extension for SSHing, the *.lan needs to be replaced in multiple places.

I'm not on a computer right now to correct it but it is a quick fix: replacing the *.lan with *.local everywhere it is located by searching with control/command (depending on operating system) for `.lan`.

Thank you again for the very nice work!

**Comment by danamlewis** [comment]
Good catch! I thought I got most of them, but found a few tucked away in various places - all .lan fixed  to .local now, thanks @diabeticgonewild!

**Comment by channemann** [comment]
Good stuff. I have a few notes and touch-ups; certainly not all need to be addressed in this PR.

https://github.com/openaps/docs/pull/4/files#diff-143c9a69795855e2aff56388ae86ae53R3
I would change the wording here; the lack of cables doesn't make it headless, it's the lack of a display.

The numbering system breaks the markdown formatting once you hit 3 because of the paragraph breaks. Might be best to move away from a formatted numbered list.

https://github.com/openaps/docs/pull/4/files#diff-143c9a69795855e2aff56388ae86ae53R51 (and similar)
If the user's SSID or password has a special character, an escape will be required; alternatively, I believe you can use single ticks (') instead of double ticks (") to enclose those two parameters.

https://github.com/openaps/docs/pull/4/files#diff-143c9a69795855e2aff56388ae86ae53R74
The user doesn't actually need to set their phone up as a hotspot to do this (which is alluded to later). It is good to test all of the networks they added, however, so testing both the iOS hotspot and the other wifi networks (home, etc) would be good.

https://github.com/openaps/docs/pull/4/files#diff-143c9a69795855e2aff56388ae86ae53R79
Probably want to expand on this a bit.

**Comment by danamlewis** [comment]
Awesome comments.  I spent 20 minutes battling with the list formatting...almost called it quits and called in the troops @bewest but thought I got it working. Guess not. Ben, help please! Or suggest better formatting to use in lieu of the lists I botched?

Re: SSID section, I was able to get it to run with double ticks to enclose the parameters. 

Would love for you to make any of those additional adds or mods per your comments, those all sound good to me.

**Comment by channemann** [comment]
If it makes you feel better, I struggled with that exact list when I was first putting it all into a gitbook as well (and eventually just gave up). I'm sure we can come up with something less crazy.

I recall having difficulty with a work password that started with a ! when using the double ticks, but perhaps I had something else messed up as well.

I'll roll the other stuff into a future edit.

-----
Chris Hannemann
chris.hannemann@gmail.com

**Comment by scottleibrand** [comment]
Yes, we should encourage people to use single quotes (') instead of double
quotes (") to enclose passwords, as some characters (like ! and others)
have special meaning at the linux command line and will not be properly
passed on to the password script if they're only enclosed in double
quotes.  That's a really annoying problem to troubleshoot, so single quotes
are safest.

**Comment by danamlewis** [comment]
@bewest ping for reviewing this PR, so @alimazaheri can give to Ed to test after it's merged into docs.

**Comment by msrcgm** [comment]
I'll have my pi tomorrow and will attempt the set up Saturday night.

Sent from my iPhone

On Sep 10, 2015, at 10:00 PM, Dana Lewis <notifications@github.com<mailto:notifications@github.com>> wrote:


@bewest<https://github.com/bewest> ping for reviewing this PR, so @alimazaheri can give to Ed to test after it's merged into docs.

File: docs/Setup/rpi.md [hunk 1]
```diff
@@ -40,59 +38,73 @@ wpa-psk <your-password>
...
+
+**3b. For Mac Users: Configuring WiFi Settings**
+<br>First boot your Pi. (Put the SD card into the RPi2. Plug in the compatible USB WiFi adapter into a RPi2 USB port. Get a micro USB cable and plug the micro USB end into the side of the RPi2 and plug the USB side into the USB power supply.)
 
-**Mac OS X:** Make sure that the computer is connected to the same WiFi router that the RPi2 is using.
+You cannot do 3a on a Mac, aka access EXT4 partitions without using 3rd party software. The easiest alternative it is to a) get a console cable (use [this guide](https://learn.adafruit.com/downloads/pdf/adafruits-raspberry-pi-lesson-5-using-a-console-cable.pdf)) or b) temporarily connect RPi to a router with an ethernet cable, SSH in (see below), and continue setting things up as described below (here in 3b) to get the wifi running. The below method will help you set up two or more wifi networks. This is highly recommended so you can add your home wifi network and your phone's hotspot network to use on the go.
 
-Open Terminal and enter this command:
+Do `sudo bash` and hit enter
```
Comment (bewest) [inline docs/Setup/rpi.md]: Maybe worth considering, a lot of docs use verbiage "type" instead of "do"?

**Comment by bewest** [comment]
Looks pretty good, lot of nice tweaks and fixes, thanks!
Looks good to me, please merge. :-)

**Comment by msrcgm** [comment]
File should I attempt? Or wait for further revision.

Sent from my iPhone

On Sep 13, 2015, at 1:29 PM, Dana Lewis <notifications@github.com<mailto:notifications@github.com>> wrote:


Merged #4<https://github.com/openaps/docs/pull/4>.

**Comment by danamlewis** [comment]
Ready for you to try. This view is probably easiest to look at:
http://openaps.gitbooks.io/building-an-open-artificial-pancreas-system/content/docs/Setup/rpi.html

On Sun, Sep 13, 2015 at 1:35 PM, msrcgm <notifications@github.com> wrote:

> File should I attempt? Or wait for further revision.
>
> Sent from my iPhone
>
> On Sep 13, 2015, at 1:29 PM, Dana Lewis <notifications@github.com<mailto:
> notifications@github.com>> wrote:
>
>
> Merged #4<https://github.com/openaps/docs/pull/4>.
>
> —
> Reply to this email directly or view it on GitHub<
> https://github.com/openaps/docs/pull/4#event-408141859>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openaps/docs/pull/4#issuecomment-139915231>.
>



-- 
Dana Lewis | http://www.twitter.com/danamlewis |
http://www.linkedin.com/in/danalewis
"Doing something for someone else is more important than anything you would
do for yourself."

**Pull request merged**